### DISCUSSION TITLE
Copilot Chat version discrepancy

### DISCUSSION THREAD

**Discussion by nick-youngblut** [community/community#82742] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

The latest version of GitHub Copilot Chat, according to the [version history](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat&ssr=false#version-history) is 0.12.2023122001, but I only see v0.7.1 in the VS Code extension info.

<img width="775" alt="Screenshot 2023-12-27 at 7 22 00 PM" src="https://github.com/community/community/assets/2468572/01162b50-fdf7-446b-b3c1-e6d9e8785cd6">

<img width="593" alt="Screenshot 2023-12-27 at 7 22 07 PM" src="https://github.com/community/community/assets/2468572/26f76313-cc91-4dd9-b22e-f9d7ffebbf05">

**Comment by harrylal** [comment]
Hi @nick-youngblut , You could switch to pre-release version to get the mentioned 0.12.2023122001
![image](https://github.com/community/community/assets/42095702/0131494c-e89a-4db9-a598-149ea989a065)

**Reply by nick-youngblut** [reply]
Thanks @harrylal! I must have missed that info in the docs. There is info in the docs about switching between Preview and Pre-release, correct?

**Reply by harrylal** [reply, marked as answer]
@nick-youngblut  Yup! , you can find details in the [docs](https://code.visualstudio.com/docs/editor/github-copilot#_are-there-prerelease-builds-of-the-copilot-extensions) and also at #59964.
